packages/veto-app/frontend/src/api/index.ts

```diff
@@ -1,14 +1,7 @@
 import axios from 'axios'
 
-export const getNewAddress = async (): Promise<string> => {
-  const { data }: { data: { address: string } } = await axios(`http://${window.location.host}/api/getnewaddress`)
-  return data.address
-}
-
-export const getBalance = async (): Promise<string> => {
-  const { data }: { data: { balance: string } } = await axios(`http://${window.location.host}/api/getbalance`)
-  return data.balance
-}
+export const getBalanceUrl = `http://${window.location.host}/api/getbalance`
+export const getNewAddressUrl = `http://${window.location.host}/api/getnewaddress`
```

Comment:
Is it worth keeping this data in here anymore? Seems like its only used in one place, will it be used anywhere else in the future?